index.js

```diff
@@ -117,12 +132,19 @@ class Dalai {
     }
   }
   async query(req, cb) {
+    console.log(`> query:`, req)
```

Comment:
Very useful.  You might consider logging the model's output to the console, too.